### PR TITLE
Feat add set bit false

### LINKED ISSUE #440: feat: add `false` to set_bit
We can use bitwise not `~` on u256.

https://github.com/snapshot-labs/sx-starknet/blob/687401f87a391c8203d75a261a27ba7639b92f92/starknet/src/utils/bits.cairo#L21

### PR DESCRIPTION
**Pull request by pscott**
Uses the `~` operator on u256.
Adds the `_remove_voting_strategies` function.

Closes #440

### CODE REVIEW

File: starknet/src/space/space.cairo [hunk 1]
```diff
@@ -664,14 +665,26 @@ mod Space {
             i += 1;
         };
         _active_voting_strategies::write(cachedActiveVotingStrategies);
+        let mut i = 0_usize;
```
Comment (Orland0x) [inline starknet/src/space/space.cairo]: typo line?

File: starknet/src/space/space.cairo [hunk 2]
```diff
@@ -664,14 +665,26 @@ mod Space {
...
         _next_voting_strategy_index::write(cachedNextVotingStrategyIndex);
     }
-    // TODO: need to impl set_bit to false first
-    // fn _remove_voting_strategies(_voting_strategies: Array<Strategy>) {
-    //     let index = next_voting_strategy_index::read();
-    //     voting_strategies::write(index, _voting_strategy);
-    //     next_voting_strategy_index::write(index + 1_u8);
-    // }
+
+    fn _remove_voting_strategies(_voting_strategies: Array<u8>) {
```
Comment (Orland0x) [inline starknet/src/space/space.cairo:671]: we should probably add a check for at least one voting strategy like https://github.com/snapshot-labs/sx-evm/blob/6c7830f9466ad4f6324afa6b4e02fc3818791162/src/Space.sol#L384C5-L384C5

File: starknet/src/space/space.cairo (hunk 1 above)
Comment (pscott) [reply starknet/src/space/space.cairo]: 8613657e2f8a09bd19bfc48e2cc4163ceb588095

File: starknet/src/space/space.cairo (hunk 2 above)
Comment (pscott) [reply starknet/src/space/space.cairo:671]: 07a8bb94fb0a12261d6e6f270878f10dd2e86833

**Review by Orland0x — APPROVED** [review]
uTACK

**Pull request merged**